src/components/doc-explorer/OtherSearchResults.tsx

```diff
@@ -1,16 +1,17 @@
 import * as React from 'react';
 
 import { isMatch, highlightTerm } from '../../utils';
+import Markdown from '../utils/Markdown';
 
-interface SearchResultsProps {
+interface OtherSearchResultsProps {
```

Comment:
Please separate this and other fixes improvements refactoring into separate PRs.
It will make it easier to review this one.

Reply:
Done. Moved into separate pull request #159.